pypd/models/incident.py

```diff
@@ -100,6 +100,116 @@ def reassign(self, from_email, user_ids):
                               data=data,)
         return result
 
+    def add_responders(self, from_email, requester_id, message, user_ids=None, escalation_policy_ids=None):
+        """Request responders to join an incident using a valid email address."""
+        endpoint = '/'.join((self.endpoint, self.id, 'responder_requests'))
+
+        if from_email is None or not isinstance(from_email, six.string_types):
+            raise MissingFromEmail(from_email)
+
+        if not (user_ids or escalation_policy_ids):
+            raise InvalidArguments('Need at least one target to be supplied')
```

Comment:
Let's make this more clear and say `Need at least one user ID or escalation policy ID to be supplied`